examples/AeroSharp.Examples/Maps/MapGetByRankExample.cs

```diff
@@ -0,0 +1,47 @@
+using AeroSharp.Connection;
+using AeroSharp.DataAccess;
+using AeroSharp.DataAccess.MapAccess;
+using AeroSharp.Examples.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Aerospike.Client;
+
+namespace AeroSharp.Examples.Maps;
+
+public class MapGetByRankExample : IExample
+{
+    private const string SetName = nameof(MapGetByRankExample);
+
+    private readonly IMap<string, string> _map;
+
+    public MapGetByRankExample(IClientProvider clientProvider)
+    {
+        var dataContext = new DataContext(ExamplesConfiguration.AerospikeNamespace, SetName);
+        var mapConfiguration = new MapConfiguration()
+        {
+            Order = MapOrder.KEY_VALUE_ORDERED
+        };
```

Comment:
Does `GetByRank` for maps assume order? Is it possible that if `MapConfiguration.Order` is not configured properly (set to unordered during `PUT` operations, etc), `GetByRank` could return an unexpected result? 

If so, should we add documentation on this?